README.md

```diff
@@ -41,22 +41,19 @@ The Wukong CLI is a set of tools to manages Mindvalley DevOps resources. Its goa
 
 ## IMPORTANT !!
 
-The latest release of Wukong CLI contains several breaking changes. Users must upgrade to the latest version, which is at least `0.0.4-beta1` in order to continue using the Wukong CLI. If you are using Wukong version < `0.0.4-beta1`, please follow the below instructions:
+The latest release of Wukong CLI contains several breaking changes. Users must upgrade to the latest version, which is at least `0.0.5-beta2` in order to continue using the Wukong CLI. If you are using Wukong version < `0.0.5-beta2`, please follow the below instructions:
 
 ```
 # Check the current version of Wukong CLI.  
 $ wukong --version
 
-# If your version is < 0.0.4-beta1.  
+# If your version is < 0.0.5-beta2.  
 $ brew update  
 $ brew upgrade wukong  
 
-# Confirm that you're on version >= 0.0.4-beta1.  
+# Confirm that you're on version >= 0.0.5beta2.  
 $ brew list --versions wukong
-wukong 0.0.4-beta1
-
-# Delete the current config.  
-$ rm ~/.config/wukong/config.toml
+wukong 0.0.5-beta2
 
 # Re-init the Wukong CLI.  
 $ wukong init
```

Comment:
I believe the whole section is not needed as the `wukong` should be able to handle the config file breaking changes. 
Or we could just say:

# IMPORTANT !!

Since the Wukong CLI is not stable yet (not v1.0.0), users must always use the latest version. The latest version is **0.0.5-beta2**. 
If you are using Wukong version < **0.0.5-beta2`**, please follow the below instructions:

```bash
# Check the current version of Wukong CLI.  
$ wukong --version
# If your version is < 0.0.5-beta2.  
$ brew update  
$ brew upgrade wukong  
```

Reply:
good idea. I updated the README with your recommendation .